lib/active_hash/base.rb

```diff
@@ -50,6 +50,25 @@ def field_names
         @field_names ||= []
       end
 
+      #
+      # Useful for CSV integration needing column names as strings.
+      #
+      # @return [Array<String>] An array of column names as strings.
+      #
+      # @example Usage
+      #  class Country < ActiveHash::Base
+      #    fields :name, :code
+      #  end
+      #
+      #  Country.column_names
+      #  # => ["id", "name", "code"]
+      #
+      def column_names
+        field_names.map do |field|
+          field.respond_to?(:name) ? field.name : field.to_s.freeze
+        end
```

Comment:
Now field names are all symbols, so I think we can simplify this:

```suggestion
        fields.map(&:name)
```

Does that work for you?

Reply:
After merging the latest master, the code worked perfectly! https://github.com/active-hash/active_hash/pull/311/commits/030685854a7876130595b630c283fbba7265660e